src/xdocs/config.xml

```diff
@@ -952,30 +957,18 @@
 
     </section>
 
-    <section name="XML Structure">
+    <section name="XML Configuration Structure">
 
-      <b>Configuration XML Document</b>
       <p>
```

Comment:
Shouldn't this and the next paragraph be merged? They are talking about the same thing.

Reply:
first paragraph was removed.